src/instance/ContainerInstance.ts

```diff
@@ -50,9 +50,9 @@ export default class ContainerInstance implements Searchable, WithAttributes {
     }
   }
 
-  public toJSON(camelCase = false, convert = true): IContainerJSON {
+  public toJSON(camelCase = false, convert = true, shouldSkip = (ins: Instance) => false): IContainerJSON {
```

Comment:
I would create a type for `(ins: Instance) => false` so you don't have to duplicate the type definition all the way down.  I also can't decide whether you should just restrict it to ListChildInstance as the input here since thats all we really check...  I could see an argument either way since in the future we may check other things.